src/Stryker.Core/Stryker.Core/Reporters/HtmlReporter/HtmlReporter.cs

```diff
@@ -32,12 +36,19 @@ public void OnAllMutantsTested(IReadOnlyProjectComponent reportComponent)
 
             WriteHtmlReport(reportPath, mutationReport.ToJsonHtmlSafe());
 
-            var clickablePath = reportPath.Replace("\\", "/");
-            clickablePath = clickablePath.StartsWith("/") ? clickablePath : $"/{clickablePath}";
+            var clickablePath = FilePathUtils.NormalizePathSeparators(reportPath);
+            clickablePath = clickablePath.StartsWith(Path.DirectorySeparatorChar) ? clickablePath : Path.DirectorySeparatorChar + clickablePath;
 
             _consoleWriter.Write(Output.Green($"\nYour html report has been generated at: \n " +
                 $"file://{clickablePath} \n" +
                 $"You can open it in your browser of choice. \n"));
+
+            if (_options.ReportTypeToOpen == Options.Inputs.ReportType.Html)
+            {
+                clickablePath = clickablePath.Remove(0, 1);
+
+                _processWrapper.Start("file://" + clickablePath);
+            }
```

Comment:
```suggestion
            var normalizedPath = FilePathUtils.NormalizePathSeparators(reportPath);
            var clickablePath = normalizedPath.StartsWith(Path.DirectorySeparatorChar) ? normalizedPath : Path.DirectorySeparatorChar + normalizedPath;

            _consoleWriter.Write(Output.Green($"\nYour html report has been generated at: \n " +
                $"file://{clickablePath} \n" +
                $"You can open it in your browser of choice. \n"));

            if (_options.ReportTypeToOpen == Options.Inputs.ReportType.Html)
            {
                _processWrapper.Start("file://" + normalizedPath);
            }
```

Perhaps we should first store a normalized path in a variable, and then make it a clickable path, because now on line 48 we are removing the extra seperator character again to make it a file open uri.